conclave-host/src/main/kotlin/com/r3/conclave/host/internal/gramine/GramineEnclaveHandle.kt

```diff
@@ -71,6 +72,13 @@ class GramineEnclaveHandle(
         return process.inputStream.reader().use { it.readText() }.trimEnd()
     }
 
+    // ProcessBuilder.Redirect.DISCARD
+    // was introduce in Java 9 but Enclave still supports Java 8.
+    // That is why NULL_FILE was added in. The code was copied from ProcessBuilder.Redirect.DISCARD
+    private val NULL_FILE: File = File(
+        if (System.getProperty("os.name")
+                .startsWith("Windows"))"NUL" else "/dev/null"
```

Comment:
Also, I am confused. How are we going to run Gramine in Windows? How?

Reply:
This block of code was copied from the Java library. It should not be changed so people know that it can be replaced by `Redirect.DISCARD`.

Reply:
My opinion is that a comment is enough and we should not mention Windows here.